articles/azure/azs-how-use-azure-api.md

```diff
@@ -36,73 +38,95 @@ To obtain an access token:
+| \$ClientID     | The application ID to authenticate against the Azure Stack Hub API   | <form oninput="result.value=clientid.value;result2.value=clientid.value" id="clientid" style="display: inline;"><input type="text" id="clientid" name="clientid" style="display: inline;" placeholder="1950a258-227b-4e31-a9cf-717495945fc2"/></form> |
 
-    > [!NOTE]
-    > Format the request body using the Content-Type `x-www-form-urlencoded`.
+### [Bash](#tab/tabid-1)
 
-3. An example request might look something like the following
+<pre><code class="language-bash"># Send POST request to Azure REST authentication/login endpoint to retrieve access token. 
+curl -X "POST" "https://login.microsoftonline.com/<output form="tenant" name="result" style="display: inline;">contoso.onmicrosoft.com</output>/oauth2/token" \
+-H "Content-Type: application/x-www-form-urlencoded" \
+--data-urlencode "client_id=<output form="clientid" name="result" style="display: inline;">1950a258-227b-4e31-a9cf-717495945fc2</output>" \
+--data-urlencode "grant_type=password" \
+--data-urlencode "username=<output form="azsusername" name="result" style="display: inline;">user</output>@<output form="tenant" name="result2" style="display: inline;">contoso.onmicrosoft.com</output>" \
+--data-urlencode "password=<output form="azspassword" name="result" style="display: inline;">Password123!</output>" \
+--data-urlencode "resource=https://management.as2ukcloud.onmicrosoft.com/90ada28c-5aed-4248-90c7-0538504217f1"
+</code></pre>
 
-    ```bash
-    curl -X "POST" "https://login.windows.net/160f539f-8571-4c96-9361-797645c24e75/oauth2/token" \
-    -H "Content-Type: application/x-www-form-urlencoded" \
-    --data-urlencode "client_id=1950a258-227b-4e31-a9cf-717495945fc2" \
-    --data-urlencode "grant_type=password" \
-    --data-urlencode "username=admin@ukcloud.onmicrosoft.com" \
-    --data-urlencode 'password=Password12345' \
-    --data-urlencode "resource=https://management.ukcloud.onmicrosoft.com/c0a3a144-57ae-42e2-a5da-19e945a25deb"
+### [PowerShell](#tab/tabid-2)
 
-4. If the authentication is successful, the endpoint returns an access token. For example:
+<pre><code class="language-PowerShell"># Declare variables
+$Tenant = "<output form="tenant" name="result3" style="display: inline;">contoso.onmicrosoft.com</output>"
+$UserName = "<output form="azsusername" name="result2" style="display: inline;">user</output>@$Tenant"
+$UserPassword = "<output form="azspassword" name="result2" style="display: inline;">Password123!</output>"
+$AuthRequestBody = @{
+    "grant_type" = "password"
+    "client_id" = "<output form="clientid" name="result2" style="display: inline;">1950a258-227b-4e31-a9cf-717495945fc2</output>"
+    "resource" = "https://management.as2ukcloud.onmicrosoft.com/90ada28c-5aed-4248-90c7-0538504217f1"
```

Comment:
Discuss URL with @cblackuk on Monday. Should we keep it? For ease of use? Or remove?